scripts/rewrite/rewrite-snapshots.js

```diff
@@ -66,13 +82,12 @@ let history;
 
     ({ serviceId, documentType } = renamer.applyRules(serviceId, documentType));
 
-    const { id: snapshotId } = await history.recordSnapshot({
+    const { id: snapshotId } = await recorder.recordSnapshot({
       serviceId,
       documentType,
       content,
       mimeType,
-      authorDate: commit.date,
-      extraChangelogContent: commit.body,
```

Comment:
> We could have a generic, “storage adapter”-level metadata storage mechanism that would store them as properties in MongoDB and as a text list in Git commit messages.
> But there is no use case for that at the moment 😀

Reply:
I had the same idea and came to the same conclusion! :D